package.json

```diff
@@ -62,5 +67,7 @@
     }
   ],
   "repository": "mozilla-b2g/mozilla-raptor",
-  "scripts": {}
+  "scripts": {
+    "test": "gulp"
```

Comment:
`test` will be reserved for running unit tests, so let's add a new target: `"lint": "gulp lint"`.